### ISSUE TITLE
wsbind rules deactivate when dpms off is used in a multi monitor configuration

### ISSUE THREAD

**Issue by Erothas** [hyprwm/Hyprland#1867]
> steps to reproduce

1. Assign different workspaces to each monitor. Say 1-5 to first monitor, 6-10 to second monitor.
2. Simply use any standard idle command through swayidle, like what I use: `swayidle -w timeout 300 'swaylock -f -c 000000' timeout 600 'hyprctl dispatch dpms off' resume 'hyprctl dispatch dpms on' before-sleep 'swaylock -f -c 000000'`
3. Let the dpms switch off the monitors.
4. Move mouse/click on keyboard to wake up monitors.

> Expected outcome

All the open windows and applications to retain their position in each respective workspace and monitor.

> Noted outcome

All aforementioned open windows and applications shift to second monitor, with every workspace -including the ones assigned to first monitor- shifting to the second monitor when they are not supposed to.

--------------------------------------------------------------------------------------------------------------------------------------------

So, basically every time monitors go to sleep. I wake them up, and I have to free up workspace 1-5, then reopen them on the first monitor again, and relaunching/moving around a lot of stuff back. The entire process takes way too long, and should not happen in the first place.

It seems like Hyprland sends all workspaces to the active monitor in case the other is disconnected. Sway does not do that. Which is why it has never been a problem there. Sway will permanently sticky the assigned workspaces to each respective monitor, even in the case of one being disconnected.
